src/sentry/api/serializers/models/release.py

```diff
@@ -62,17 +62,19 @@ def get_users_for_authors(organization_id, authors, user=None):
     }
```

Comment:
Now that I think about this more, I'm wondering if we're being too granular. Would it be better to just cache the author -> user result (ie the values in `results`), and then perform all this work if there's no lookup?

It'd look like 
```
cache_keys = ['a_to_u:<org_id>:<author_email_hash>' for author in authors]
fetched = cache.get_many(cache_keys)
missed = []
for serialized_user, author in zip(fetched, authors):
    if serialized_user is None:
        missed.append(author)
    result[...] = ...

if missed:
    serialized_users = <do serialization logic just for missed rows>
    cache.set_many({'a_to_u:<org_id>:<author_email_hash>': serialized_user for serialized_user, author in zip(serialized_users, missed})
```

This is pretty rough but the general idea. Then you don't need to worry about the intermediate steps, since all we care about here is converting an author to a serialized user.

Reply:
This feels like a really good solution. Going to try getting a working solution using this idea.

Reply:
Just pushed up a solution using this idea.